api/envoy/config/listener/v3/listener_components.proto

```diff
@@ -109,6 +109,131 @@ message FilterChainMatch {
     EXTERNAL = 2;
   }
 
+  // Matches filter chains by the destination port.
+  // [#next-free-field: 6]
+  message DestinationPort {
```

Comment:
Should the two fields in this proto be in a `oneof`?  Or are there cases where they will both be used at the same time?

Reply:
Okay.  Please make that clear in the comments.